src/ProducerEventProcessor/ProducerEventProcessor.cpp

```diff
@@ -270,6 +321,11 @@ void ProducerEventProcessorImpl::ProcessCallstackSampleAndTransferOwnership(
 
 void ProducerEventProcessorImpl::ProcessCaptureFinishedAndTransferOwnership(
     CaptureFinished* capture_finished) {
+  if (!thread_state_slice_tid_and_begin_timestamp_to_callstack_id_.empty()) {
+    ORBIT_ERROR(
+        "There are saved callstacks for thread state slices left being not merged to the slice "
+        "after the capture finished.");
```

Comment:
```suggestion
        "Some saved callstacks for thread state slices are left not merged to any slice after the capture finished");
```

Just trying to simplify this sentence a bit, it feels a bit heavy.

Reply:
Sounds good.